### ISSUE TITLE
dpdk-testpmd test under the io mode no tx and rx are both 0

### ISSUE THREAD

**Issue by liyangyang-linux** [missinglinkelectronics/dpdk-stable#5]
Hello, when I use dpdk-testpmd for testing, there is no data and throughput for tx and rx in io mode, but tx is OK when I set it to txonly. May I ask what may be the cause of this? I am looking forward to your answer very much

**Comment by pmo73** [comment]
Hello, I am currently unable to reproduce the error. Can you provide more information or output? Does the driver work with the l2fwd example? I also tested with this at the time alongside the testpmd app.

**Comment by pmo73** [comment]
I have tested the driver with the corundum design on the git-hash given in the README on Ubuntu 22.04 with kernel version 5.15.0-89 and l2fwd example and testpmd app also work for me with switching to txonly|rxonly for dpdk 22.11

**Comment by liyangyang-linux** [comment]
Thank you very much for your answer. After replacing the firmware of the fpga, it can work normally. The submitted version you provided is very useful

**Comment by thenveerpk** [comment]
Hi @pmo73 ,

I built the Design using the master branch (Git hash: 1ca0151b), but I’m facing an issue where no packets are being transmitted or received (RX/TX).

Could you please advise on what changes are needed on the driver side to get it working correctly?

Thank you!

**Comment by pmo73** [comment]
Hi @thenveerpk,
I apologize for the delay in replying, but I didn't have time to take a closer look beforehand. I looked at the commit and it is much newer than the commit I tested it against back then. For the exact details I would like to refer you to the README in the driver: https://github.com/missinglinkelectronics/dpdk-stable/blob/22.11-add-corundum-mqnic-pmd/drivers/net/mqnic/README.md
Is it possible for you to switch to the older version? Unfortunately, I no longer have access to a board and the setup, so I can't retest against a newer version or publish a newer one.
Kind regards
